index.html

```diff
@@ -0,0 +1,125 @@
+        <img src="./assets/svg/icon-twitter.svg" alt="twitter icon">
+        <a href="#">English</a>
+        <a href="#">My page</a>
+        <a href="#">Log out</a>
+      </div>
+    </div>
+    <nav id="deskNav">
+      <a href="#"><img src="./assets/png/Logo-color.png" alt=""></a>
+      <a href="./about.html">About</a>
+      <a href="#">Program</a>
+      <a href="#">Join</a>
+      <a href="#">Sponsor</a>
+      <a href="#">News</a>
+      <button type="button">Tickets now!</button>
+    </nav>
+    <section id="heroSection">  
+      <div id="backgroundDesk">  
+        <div class="hero-content">
+          <h3>"Hey! Virtual world"</h3>
+          <img src="./assets/png/hero-title.png" alt="">
+          <div id="sumContainer">
+            <p>
+              Launched in the year 2000, COOLors fest is dedicating itself to the presentation of live electronic music and real-time audiovisual performance, making it one of the rare outlets in North America for such innovations.
+            </p>
+          </div>  
+          <h2>2021.08_05(THU)~ 16(FRI)</h2>
+          <p>@ National Museum of Quebec, Art Center Nabi and more.</p>
+        </div>
+    </div>
+    </section>
+    <main>
+      <section id="mainProgram">
+        <h2>Main Program</h2>
+        <div class="deco"></div>
+        <ul id="actContainer">
+          <li class="actCard">
+            <img src="./assets/png/icons8-concert-50.png" alt="decoration icon">
+            <h4>Concerts</h4>
+            <p>Listen to your favorite artist from various contries. no matter where you are.</p>
+          </li>
+          <li class="actCard">
+            <img src="./assets/png/icons8_vr-cardboard.png" alt="decoration icon">
+            <h4>VR Exhibition</h4>
+            <p>Between creation and exploration, and reflection and innovation inside a multiplatform event</p>
+          </li>
+          <li class="actCard">
+            <img src="./assets/png/icons8-performance-50.png" alt="decoration icon">
+            <h4>Forum</h4>
+            <p>COOLors unveils the Festival’s in-venue programming as well as the first experts who will take part in the Forum</p>
+          </li>
+          <li class="actCard">
+            <img src="./assets/png/icons8-entypo_ticket.png" alt="decoration icon">
+            <h4>Nocturne series</h4>
+            <p>These 12 performances, most of which are world premieres, showcase the talents of about twenty artists.</p>
+          </li>
+          <li class="actCard">
+            <img src="./assets/png/icons8-music-festival-50.png" alt="decoration icon">
+            <h4>XR Salon</h4>
+            <p>This year’s XR Salon will also put into focus the circulation and distribution of XR works.</p>
+          </li>
+        </ul>
+        <button type="button" id="mainSBtn" class="deskNone">Join CC Global Summit 2015</button>
+        <button type="button" class=""></button>
+      </section>
+      <section id="featSpeakers">
```

Comment:
- [ ] **The design for the featured speaker section doesn't match the original design. Please fix it**

Original Design|Cloned Design
:-------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/125125341-10f63380-e117-11eb-8572-6fed76c15720.png)|![image](https://user-images.githubusercontent.com/59168430/125125390-24a19a00-e117-11eb-845d-f4db8aec7f3c.png)

The overall spacing between text and images is significantly different on the cloned version. Please modify it so that it appears as close as possible to the original design.

Reply:
Fixed :heavy_check_mark: